### ISSUE TITLE
Image is not defined

### ISSUE THREAD

**Issue by talentlessguy** [ai/webp-in-css#15]
When I try to build my site using [Poi](https://poi.js.org), PostCSS and webp-in-css I get the following error:

```sh
./styles/footer.css
Module build failed (from ./node_modules/mini-css-extract-plugin/dist/loader.js):
ModuleBuildError: Module build failed (from ./node_modules/postcss-loader/src/index.js):
Error: Loading PostCSS Plugin failed: Image is not defined

(@/home/v1rtl/Desktop/Coding/v1rtl.site/.postcssrc)
    at Array.map (<anonymous>)
```

As I understand this is because my site uses node.js environment and not browser one and that's why `Image` is not available. How I can fix this?

My `.postcssrc`:

```json
{
  "plugins": 
  {
    "precss": {},
    "autoprefixer": {},
    "webp-in-css": {},
    "cssnano": {
      "preset": "default"
    }
  }
```

One of styles that uses webp:

```css
header {
  display: flex;
  align-items: center;
  flex-direction: column;
  user-select: none;
  height: 100vh;
  background-image: url('../assets/background.webp');
}
```

If you need to check all the files, repo is [here](https://github.com/talentlessguy/v1rtl.site)

**Comment by ai** [comment]
Replace `webp-in-css` to `webp-in-css/plugin`

**Comment by talentlessguy** [comment]
> Replace `webp-in-css` to `webp-in-css/plugin`

oops, sorry. I'll try it and close if it works :D

**Comment by ai** [comment]
Don't close it. It is not your fault, wet need to improve DX by warning.

**Comment by talentlessguy** [comment]
okay :ok_hand:

**Comment by ai** [comment]
I renamed `index.js` to `polyfill.js` to avoid this mistake in the future
https://github.com/ai/webp-in-css/commit/e4f6959e6e3c151d694bdb1cd51c5c96bb707f34

**Comment by ai** [comment]
The fix was released in 0.3
